meson.build

```diff
@@ -110,10 +119,8 @@ pkgconf_exe = executable('pkgconf',
 if get_option('tests')
   kyua_exe = find_program('kyua')
   atf_sh_exe = find_program('atf-sh')
-  test('kyua', kyua_exe, args : ['--config=none', 'test', '--kyuafile=' + join_paths(meson.build_root(), 'Kyuafile'), '--build-root=' + meson.build_root()])
-
-
-  configure_file(input : 'Kyuafile.in', output : 'Kyuafile', configuration : cdata)
+  kyuafile = configure_file(input : 'Kyuafile.in', output : 'Kyuafile', configuration : cdata)
```

Comment:
May want to match the `:` style with the rest of the files. Seem to be attached to the key elsewhere.

Reply:
The style is all over the place here, and I'm inclined to leave it alone since all I did was assigned the output.